.github/workflows/trunk.yml

```diff
@@ -174,3 +174,23 @@ jobs:
           { config: "default", shard: 6, num_shards: 6, runner: "windows.g5.4xlarge.nvidia.gpu" },
           { config: "force_on_cpu", shard: 1, num_shards: 1, runner: "windows.4xlarge.nonephemeral" },
         ]}
+
+  linux-jammy-py3-clang12-executorch-build:
+    name: linux-jammy-py3-clang12-executorch
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: linux-jammy-py3-clang12-executorch
+      docker-image-name: pytorch-linux-jammy-py3-clang12-executorch
+      test-matrix: |
+        { include: [
+          { config: "executorch", shard: 1, num_shards: 1, runner: "linux.2xlarge" },
+        ]}
+
+  linux-jammy-py3-clang12-executorch-test:
```

Comment:
Are those merge-blocking jobs? Where is it set?

Reply:
Yup, all the jobs in trunk are merge blocking.  In PyTorch, we are using custom code here https://github.com/pytorch/pytorch/blob/main/.github/scripts/trymerge.py#L1919 to handle the merge, so it's fairly complex.  My plan is to refactor this script to cover ExecuTorch too.